content/en/user-guide/aws/rds/index.md

```diff
@@ -129,3 +135,42 @@ Please consider the following notes regarding default usernames/passwords and da
 - You can use any `master-username`, except "postgres", for creating a new RDS instance. The user will automatically be created.
 - The user "postgres" is special, and it is not possible to create a new RDS instance with this user name.
 - Do not use `db-name` "postgres" as it is already in use by LocalStack.
+
+## IAM Authentication Support
+
+IAM auth token can be used to connect to RDS. This feature is currently only supported for Postgres in LocalStack.
+
+{{< alert title="Note" >}}
+Please be aware that the IAM authentication is not verified at this point. Meaning any user that is granted the role `rds_iam` will receive a valid token.
+{{< /alert >}}
+
+### Using IAM example
+
+The following show cases the IAM functionality for Postgres:
+
+* create a DB instance, and retrieve the host and port for the instance
+* connect to the DB using the master username and password. Then create a new user and grant it the role `rds_iam`:
+   * `CREATE USER <username> WITH LOGIN`
+   * `GRANT rds_iam TO <username>`
+* receive a token for the `<username>`:
+   * awslocal rds generate-db-auth-token --username $USER --hostname $HOST --port $PORT
+* connect to the DB with the user you have a created and the token you have received in the previous step as password
+
+{{< command >}}
+$ MASTER_USER=hello
+$ MASTER_PW=MyPassw0rd!
```

Comment:
Exclamation marks can cause issues in some shells - better wrap in single quotes. 👍 
```suggestion
$ MASTER_PW='MyPassw0rd!'
```